### PR TITLE
delete examples folder

### PR DESCRIPTION
**Pull request by thast**
straightforward delete for issue #61 . @lheagy ?

### CODE REVIEW

**Comment by lheagy** [comment]
There are a couple examples that build on this code - we can add em_examples as a dependency and import from there instead

**Comment by lheagy** [comment]
Prior to merging 

@sgkang, can you confirm if the following notebooks have been included in the em_examples repo and are ok to delete from here: 
- [ ] examples/geophysical_surveys/emi_tutorial/InducedCurrents.ipynb
- [ ] examples/physical_properties/chargeability/ColeCole.ipynb
- [ ] examples/physical_properties/electrical_conductivity/ConductivityGenFigs.ipynb

File: content/maxwell2_dc/fields_from_grounded_sources_dcr/electrostatic_sphere.rst [hunk 1]
```diff
@@ -453,17 +435,8 @@ The only parameters that have changed are the radius and the conductivity of the
     electrode_spacing = 20.
     PlotOpt = 'Total'
 
-    #Initializing the figure
```
Comment (lheagy) [inline content/maxwell2_dc/fields_from_grounded_sources_dcr/electrostatic_sphere.rst]: @thast : and again here. the two_configurations_comparison code in the em_examples repository does not take ax as an input. Thanks!

**Comment by lheagy** [comment]
@thast, @sgkang : If you guys have a chance to take a look at this, it would be great to get it in before the meeting tomorrow!

**Comment by thast** [comment]
Tried it on my machine. Seems to work properly! Nice work!

**Comment by lheagy** [comment]
feel free to merge if you are happy with it @thast !

**Comment by lheagy** [comment]
@thast, @sgkang : can you give a 👍 if this is good to go? Thanks!

**Pull request merged**